--- generate_errors_orig.pl	2026-07-29 13:57:43.180344966 -0400
+++ generate_errors.py	2026-07-29 13:57:43.191627371 -0400
@@ -167,7 +167,7 @@
     {
         $code_check = \$ll_code_check;
         $old_define = \@ll_old_define;
-        $white_space = '        ';
+        $white_space = '    ';
     }
     else
     {
@@ -219,8 +219,15 @@
         ${$old_define_suffix} = $define_suffix;
     }
 
-    ${$code_check} .= "${white_space}case -($error_name):\n".
-                      "${white_space}    return( \"$module_name - $description\" );\n"
+    if ($error_name eq "MBEDTLS_ERR_SSL_FATAL_ALERT_MESSAGE")
+    {
+        # no-op, this case is hard-coded in error.fmt
+    }
+    else
+    {
+        my $error_text = $error_name =~ s/^MBEDTLS_ERR_//r;
+        ${$code_check} .= "${white_space}{ -($error_name), \"$error_text\" },\n"
+    }
 };
 
 if ($ll_old_define[0] ne "")
